docgen/src/data/communityHeader.json

```diff
@@ -42,7 +42,7 @@
       "label": "InstantSearch.js",
       "logo": "",
       "caret": "",
-      "url":"https://community.algolia.com/instantsearch.js/v2/",
+      "url":"https://community.algolia.com/instantsearch.js/",
```

Comment:
This could be a communityHeader.js and reuse the configuration to make this tailored to the env rather than static.